dbt/exceptions.py

```diff
@@ -357,3 +357,15 @@ def raise_ambiguous_alias(node_1, node_2):
             duped_name,
             node_1['unique_id'], node_1['original_file_path'],
             node_2['unique_id'], node_2['original_file_path']))
+
+
+def raise_patch_targets_not_found(patches):
+    patch_list = '\n\t'.join(
+        '{} (referenced in path {})'.format(p.name, p.path)
+        for p in patches.values()
+    )
+    raise_compiler_error(
+        'dbt could not find models for the following patches:\n\t{}'.format(
+            patch_list
```

Comment:
i wrote a big thing but now i understand that path == original_file_path. i'd recommend swapping out 'referenced in path' for 'defined in'

```
model abc (defined in models/prod/schema.yml)
```